.github/workflows/pr_push_workflow.yml

```diff
@@ -0,0 +1,62 @@
+name: PR/push
+
+on: [push, pull_request]
+
+concurrency: 
+    group: ${{ github.head_ref }}-${{ github.workflow }}
+    cancel-in-progress: true
+
+jobs:
+    Fast_build:
+        name: Fast build
+        strategy:
+            matrix:
+                os: [ubuntu-latest, windows-latest]
+                include:
+                    - os: ubuntu-latest
+                      nproc: $(nproc)
+                      pool_scalable: 'ON'
+                      jemalloc: 'ON'
+                    - os: windows-latest
+                      nproc: $Env:NUMBER_OF_PROCESSORS
+                      pool_scalable: 'OFF'
+                      jemalloc: 'OFF'
+        runs-on: ${{matrix.os}}
+
+        steps:
+        - name: Checkout repository
+          uses: actions/checkout@v4
+
+        - name: Install apt packages
+          if: matrix.os == 'ubuntu-latest'
+          run: |
+            sudo apt-get update
+            sudo apt-get install -y cmake clang libnuma-dev libjemalloc-dev libtbb-dev
```

Comment:
do you need clang for this job?

Reply:
heh, maybe CodeQL requires it, maybe not. It seems like it's redundant here. If that's the case, IMHO, it should be removed 😉

Reply:
In this FastBuild job clang is not used, should be removed.

Reply:
Removed all unnecessary packages.